packages/daimo-common/src/op.ts

```diff
@@ -167,10 +204,13 @@ export type DaimoAccountCall = {
 // the address of the claimer.
 // If the op claims a payment link, from = sender, to = claimer.
 // If the op is a swap, from = the pre-swap sender.
-export function getDisplayFromTo(op: DisplayOpEvent): [Address, Address] {
+export function getDisplayFromTo(op: TransferClog): [Address, Address] {
   if (op.type === "transfer") {
+    // TODO: is this needed for backwards compat?
```

Comment:
another backcompat question

Reply:
can delete if we do the thing described above (load a `StoredV16TransferClog` with a `preSwapTransfer` as a SwapClog)